src/Order.js

```diff
@@ -0,0 +1,15 @@
+import React, { useState } from "react";
+import RestaurantButton from "./RestaurantButton";
+
+const Order = ({ orderType }) => {
+  const [orders, setOrders] = useState(0);
+  const orderOne = () => {
+    setOrders(orders + 1);
+  };
+  return (
+    <li>
+      {orderType}: {orders} <RestaurantButton totalOrders={orderOne} />
```

Comment:
I would call this prop `orderOne`, would be more intuitive than `totalOrders`. It also indicates that it's a function.